tuned/plugins/plugin_irqbalance.py

```diff
@@ -10,7 +10,23 @@
 
 class IrqbalancePlugin(base.Plugin):
 	"""
-	Plugin for irqbalance settings management.
+	`irqbalance`::
+	
+	Plug-in for irqbalance settings management.  The plug-in
+	configures CPUs which should be skipped when rebalancing IRQs in
+	`/etc/sysconfig/irqbalance`. It then restarts irqbalance if and
+	only if it was previously running.
+	+
+	The banned (to be skipped) CPUs are specified as a CPU list via the
+	[option]`banned_cpus` option.
+	+
```

Comment:
The banned/skipped CPUs are specified as a CPU list via the [option]`banned_cpus` option.